cmd/promtool/main.go

```diff
@@ -283,9 +297,38 @@ func main() {
 	}
 }
 
+var lintError = fmt.Errorf("lint error")
```

Comment:
nit: because this is printed in the output at the start of the linting errors, it should be capitalized

```suggestion
var lintError = fmt.Errorf("Lint error")
```